### PR TITLE
Replace the player timeline

### PR DESCRIPTION
**Pull request by yostyle**
image

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageVoiceBroadcastListeningItem.kt [hunk 1]
```diff
@@ -122,10 +122,17 @@ abstract class MessageVoiceBroadcastListeningItem : AbsMessageVoiceBroadcastItem
 
     private fun bindSeekBar(holder: Holder) {
         with(holder) {
-            durationView.text = formatPlaybackTime(duration)
+            var remainingTimeText = formatPlaybackTime(duration)
+            remainingTimeView.text = if (duration < 1000) remainingTimeText else String.format("-%s", remainingTimeText)
```
Comment (jmartinesp) [inline vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageVoiceBroadcastListeningItem.kt:126]: This seems to be duplicated below, just using different inputs. Would it make sense to create a helper function for it?

**Review by jmartinesp — APPROVED** [review]
LGTM, just minor suggestion.

**Review by giomfo — APPROVED** [review]
UX review / Tests ok
@yostyle approved, please add a changelog file

**Comment by yostyle** [comment]
> UX review / Tests ok @yostyle approved, please add a changelog file

added

**Pull request merged**